grammar/abnf-grammar.txt

```diff
@@ -983,11 +983,19 @@ package-path = "*"
              / package-name "." package-path
              / "(" package-path *( "," package-path ) [","] ")"
 
+; A global const declaration consists of the 'const' keyword
```

Comment:
We are now using markdown in the ABNF documentation comments, so that the markdown is rendered in the `leo/grammar/README.md` file using Damir's converter. So it should be `<backquote>const<backquote>` instead of `<quote>const<quote>` etc. (But this can be also done in a subsequent PR.)

Reply:
I fixed this one instance for now.